### PR TITLE
Added limitations chapter to the docs

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Please let me know if I missed anything important or if this chapter should be rephrased for better clarity or anything.

### CODE REVIEW

**Comment by maehne** [comment]
@MarcinOrlowski: Could you please address the feedback that we provided to your PR?

**Comment by MarcinOrlowski** [comment]
@maehne Sure, I will fix whatever pending but that unfortunately most likely won't happen before the weekend as I am currently heavily occupied by other projects.

**Comment by maehne** [comment]
@MarcinOrlowski: Thanks for the quick reply! No worries if it takes a while. We are also currently well occupied with other things.

**Review by maehne — CHANGES_REQUESTED** [review]
@MarcinOrlowski: Could you please address the changes requested to this PR, so that it can finally be merged? Thanks!

**Comment by maehne** [comment]
@MarcinOrlowski: Could you please address the remaining review comments so that this PR can be merged in time for the upcoming version 3.8.0?

**Comment by MarcinOrlowski** [comment]
@maehne Sorry, I was pretty busy recently. I will go by all pending reviews/tasks by end of the week. Is that OK?

**Comment by maehne** [comment]
@MarcinOrlowski: If you manage to do it until the end of this week that would be great! Our target is to have a new release ready for the beginning of the autumn term mid September.

File: docs/docs.md [hunk 1]
```diff
@@ -14,6 +15,19 @@
...
 
+## Limitations ##
+
+While `Logisim-evolution` may be perfectly suitable for most of your digital circuit projects, you should
+be aware about important limitations that may affect your works:
+
+* Bidirectional pins (input or output depending on the control line state) are currently not supported by simulation engine,
+  therefore built-in library do not offer such elements, nor you will be able to contribute your own implementation as of today.
+* Each of the component of your circuit will be given equal amount of simulation time and which is not necessary what close to
+  reality where phisical characteristics of each element can be different and affect their response time. Equalizing these times
```
[the remark this answers is not in the document]
Comment (MarcinOrlowski) [reply docs/docs.md:26]: Replaced confusing sentence with @maehne's.

File: docs/docs.md [hunk 2]
```diff
@@ -14,6 +15,19 @@
 
 ---
 
+## Limitations ##
+
+While `Logisim-evolution` may be perfectly suitable for most of your digital circuit projects, you should
+be aware about important limitations that may affect your works:
```
[the remark this answers is not in the document]
Comment (MarcinOrlowski) [reply docs/docs.md:21]: Changed.

File: docs/docs.md [hunk 3]
```diff
@@ -14,6 +15,19 @@
...
+
+* Bidirectional pins (input or output depending on the control line state) are currently not supported by simulation engine,
+  therefore built-in library do not offer such elements, nor you will be able to contribute your own implementation as of today.
```
[the remark this answers is not in the document]
Comment (MarcinOrlowski) [reply docs/docs.md:24]: Made two sentences as I was unhappy with `;` separator. Not sure it's correct to start the sentence with `Therefore` though. @davidhutchens please correct if that's needed to be changed.

File: docs/docs.md [hunk 4]
```diff
@@ -14,6 +15,19 @@
...
+* Each of the component of your circuit will be given equal amount of simulation time and which is not necessary what close to
+  reality where phisical characteristics of each element can be different and affect their response time. Equalizing these times
+  prevent **discrete** circuits like flip-flops from having undetermined initial state.
+
```
[the remark this answers is not in the document]
Comment (MarcinOrlowski) [reply docs/docs.md:28]: Added.

**Review by davidhutchens — APPROVED** [review]
This looks good to me. (And there is nothing wrong with starting a sentence with "Therefore,".)

Thanks for cleaning this up. I think it will help some users better understand what they can and cannot do.

**Comment by MarcinOrlowski** [comment]
> And there is nothing wrong with starting a sentence with "Therefore,"

Thanks for the review and clarification. FYI: In some languages (i.e. Polish) it would look awkward (to natives) if you start the new sentence that way as usually you should go for compound sentence instead (hence my question).

**Review by maehne — APPROVED** [review]

**Pull request merged**